skimage/_shared/utils.py

```diff
@@ -191,3 +192,21 @@ def copy_func(f, name=None):
     return types.FunctionType(six.get_function_code(f),
                               six.get_function_globals(f), name or f.__name__,
                               six.get_function_defaults(f), six.get_function_closure(f))
+
+
+def check_random_state(seed):
+    """Turn seed into a np.random.RandomState instance
```

Comment:
It would be great if you could transform this into our doc string format. ("Parameters", "Raises", "Returns" section)